style.css

```diff
@@ -0,0 +1,922 @@
+}
+.languages img {
+  width: 3rem;
+}
+.languages-box li {
+  background: var(--languages-box);
+  width: 20%;
+  padding: 5rem;
+  margin-top: 2rem;
+  margin-right: 3rem;
+  margin-bottom: 6rem;
+}
+.languages-box img {
+  width: 60%;
+}
+.languages-box {
+  display: flex;
+}
+hr {
+  height: 1px;
+  width: 100%;
+  background: var(--icons-color);
+  margin-top: 1rem;
+}
+.skill {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 2rem;
+}
+
+.skill img {
+  width: 1.8rem;
+}
+
+.skill h4 {
+  font-size: 6rem;
+  letter-spacing: 0.1rem;
+  margin-bottom: 6rem;
+}
+/*------- About-section Ends here--------- */
+
+/*------- Contact-section starts here--------- */
+#contact-section {
+  background-image: url(./images/Contact-pg.svg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 100vh;
+  margin-top: 3rem;
+  padding-top: 10rem;
+}
+.contact-text {
+  display: flex;
+  flex-direction: column;
+  padding-top: 12rem;
+  text-align: center;
+  margin-bottom: 2rem;
+}
+
+.contact-text h2 {
+  color: #fff;
+  font-size: 12rem;
+  padding-top: 2rem;
+  margin-bottom: 6rem;
+}
+
+.contact-text p {
+  color: #fff;
+  font-size: 5rem;
+  line-height: 10rem;
+  margin-bottom: 6rem;
+  font-weight: 400;
+  letter-spacing: 1px;
+  padding-left: 26%;
+  padding-right: 26%;
+}
+
+.contact-form {
+  display: flex;
+}
+
+.form,
+.input,
+.message {
+  width: 38%;
+  font-size: 3rem;
+}
+
+.input,
+.message {
+  border-radius: 0.4rem;
+  margin-top: 20px;
+  border: 1px solid #cfd8dc;
+  font-family: "Poppins", sans-serif;
+}
+
+.input {
+  padding: 4rem;
+  border-radius: 2rem;
+  margin-bottom: 6rem;
+}
+#contact-section button {
+  background: #fff;
+}
+
+#contact-section button:hover {
+  background: var(--active-btn);
+  color: var(--tools-background);
+}
+
+.active {
+  padding: 40px 90px;
+  color: var(--tools-background);
+  border: 1px solid var(--button-text);
+  margin-top: 1rem;
+  border-radius: 2rem;
+  cursor: pointer;
+  display: flex-start;
+  font-size: 4rem;
+  background: var(--active-btn);
+}
+/* Contact-section End */
+
+/*--------------------------- Media Query ------------------------------*/
+
+/*--------------------------- Media Query ------------------------------*/
+
+@media only screen and (max-width: 768px) {
+  /* h1 {
+    font-size: 2.5rem;
+    line-height: 52px;
+    font-weight: 700;
+    color: var(--title-color);
+  }
+
+  h2 {
+    font-size: 1.9rem;
+    font-style: bold;
+    line-height: 2rem;
+  }
+
+  h3 {
+    font-size: 0.8rem;
+    font-weight: normal;
+    color: var(--year-color);
+  }
+
+  h4 {
+    font-size: 0.8rem;
+    font-weight: 500;
+  }
+
+  p {
+    font-size: 1rem;
+    line-height: 1.5rem;
+    font-weight: 200;
+    font-style: normal;
+    color: var(--content-color);
+    margin-bottom: 0.7rem;
+  }
+  header {
+    display: flex;
+    height: 2%;
+    justify-content: space-between;
+  }
+  header img {
+    width: 3rem; */
```

Comment:
- [ ] It is a better practice for HTML & CSS do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Kindly delete commited sections.